### DISCUSSION TITLE
MAME 0.270

### DISCUSSION THREAD

**Discussion by cuavas** [mamedev/discussions#145] [Announcements]
# [MAME 0.270](https://www.mamedev.org/?p=542)

We’re happy to announce that MAME 0.270 is ready!  A lot of work has gone into a lot of different areas of MAME throughout September.  One very interesting addition is the “Zoomer” PDA.  Built by Casio and marketed under multiple brand names, this PDA ran MS-DOS 3.3, GEOS 2.0 and the PenRight user interface.  It was one of the first PDAs to include software developed by Palm Computing.  Other exotic systems include the Sony NWS-3270 workstation and 68000 development boards from Marion Systems and Motorola themselves.

The Hitachi Basic Master Jr. is now working, giving a glimpse of the Japanese home computer market in the early 1980s.  UMC’s attempt at taking on Sega and Nintendo in the mid-1990s, the Super A'Can, is in a much better state than it was previously.  Although it still isn’t considered working, numerous issues with graphics and sound have been addressed, and battery-backed cartridge memory is now supported.  Several more Apple II input peripherals are now supported.  Support for hard-sectored floppy disk formats has been added, which should open up storage options for computers from the S-100 era.

The effort to understand the IGS027A CPUs and dump their internal programs is paying off.  Over a dozen slots, mahjong and card games from IGS are now playable.  If you’re a fan of these games, you can now play them in the comfort of your home with no risk of blowing your paycheque.

That’s all we’ve got time to cover here, but you can read all the exciting (and mundane) tales of development in the [whatsnew.txt file](https://www.mamedev.org/releases/whatsnew_0270.txt).  As always, you can get the source code and 64-bit Windows binary packages from [the download page](https://www.mamedev.org/release.html).

[Read the rest of this entry »](https://www.mamedev.org/?p=542#readmore)

**Comment by cuavas** [comment]
Bob Zed’s video highlighting some of the work in this release:
[![MAME 270 - What's new](https://img.youtube.com/vi/Y5wwAU_YHsU/0.jpg)](https://www.youtube.com/watch?v=Y5wwAU_YHsU)

**Comment by wyatt-wong** [comment]
Is there update on issue #141 and #142

**Reply by golieizabeleio-gmail** [reply]
We are looking for game software partners who can:
Provide standalone slot machine game software (offline, no internet required)

Ensure compatibility with physical hardware, including:

Coin insertion recognition

Coin payout control

Ticket dispensing control (via thermal printers)

Game difficulty adjustment (e.g. odds, payout rates)

Screen resolution and aspect ratio settings (for various display devices)

Customizable sound, language, and gameplay parameters

We will provide complete hardware communication protocol documentation, including interface logic and electrical standards.

🔧 Cooperation Model
The software provider delivers a burnable game software package

We will pay per burn instance (one payment per flash onto a machine)

Settlement will be conducted via anonymous and cryptocurrency methods (e.g., USDT)

Batch licensing is also supported (details negotiable)
